tests/example.spec.ts

```diff
@@ -0,0 +1,12 @@
+import { test, expect } from '@playwright/test';
+
+test('has title', async ({ page }) => {
+    await page.goto('/');
+
+    await expect(page).toHaveTitle(/Gas planner/);
+});
+
+test('has Bottom text', async ({ page }) => {
+    await page.goto('/');
```

Comment:
E2E tests execution is expensive.
Try to reuse one test setup to test multiple scenarios.
In this case we don't need to test one of these, we need to test something, which really means the page loaded successfully.